pdf_craft/pdf/handler.py

```
@@ -56,53 +56,56 @@ def pages_count(self) -> int:
+                        authors = [author_str.strip()]
+
+            modified = datetime.now(timezone.utc)
+            if metadata and "/ModDate" in metadata:
+                try:
+                    mod_date_obj = metadata["/ModDate"]
+                    if mod_date_obj:
+                        mod_date_str = str(mod_date_obj)
+                        # PDF 日期格式: D:YYYYMMDDHHmmSSOHH'mm'
+                        if mod_date_str.startswith("D:"):
+                            mod_date_str = mod_date_str[2:]
+                        # 简化处理：只取年月日时分秒
```

Ruff (0.14.8)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)